regen/mg_vtable.pl

```diff
@@ -202,8 +202,10 @@ BEGIN
                 desc => 'Extra data for restricted hashes' },
      arylen_p => { char => '@', value_magic => 1,
                    desc => 'To move arylen out of XPVAV' },
-     ext => { char => '~', desc => 'Available for use by extensions',
+     ext => { char => '~', desc => 'Variable magic available for use by extensions',
               readonly_acceptable => 1 },
+     value => { char => '^', desc => 'Value magic available for use by extensions',
```

Comment:
I can't help thinking it might want to be called `extvalue`, to point out it's for extensions (i.e. non-core) use. And maybe can the other one be renamed `extvar`, with a back-compat alias?

((I'm lately more of the opinion that things that are badly named ought to be renamed to better names with back-compat wrappers...))

Reply:
Personally I think `ext` is a terrible name/prefix that doesn't actually explain what it does (though it's still better than `uvar`)

Reply:
`PERL_MAGIC_custom(_value)`? There's precedent in `OP_CUSTOM`

Reply:
Not including 'value' definitely would be a downgrade to me, but TBH I'd prefer to keep it the way it is unless someone else also disagrees with the current name.